### PR TITLE
Add support for injecting custom styles and scripts

### LINKED ISSUE #25: Inject custom CSS
It would be useful to be able to inject custom CSS to make small overrides without having to fork the project. Something like the [docdash](https://github.com/clenemt/docdash) `scripts` config would be enough. 

Are you open to a PR for this?

### PR DESCRIPTION
**Pull request by sanbornhilland**
This allows the consumer to tweak styling and behaviour
by supplying their own custom .css and .js files to get
bundled into the docs when they're built.

Heavily influenced by the approach over at:
https://github.com/clenemt/docdash

Closes #25

### CODE REVIEW

**Comment by sanbornhilland** [comment]
@bigtimebuddy This should be backwards compatible. If it's accepted would you be open to a minor release?

BTW, Kudos to everyone who built these docs. The pixi.js docs are really easy to use.

File: README.md [hunk 1]
```diff
@@ -25,39 +25,45 @@ jsdoc -c conf.json -R README.md
...
+          "encoding": "utf8",
+          "recurse": true,
+          "private": false,
+          "lenient": true,
+          "destination": "./docs",
+          "template": "./node_modules/@pixi/jsdoc-template"
+      }
+  },
+  "pixi": {
+      "styles": ["path/to/custom/styles.css"], // Optionally include custom styling
```
Comment (bigtimebuddy) [inline README.md:64]: Could you remove the comments (they are pretty self explanatory) or change the `json` to `js` as the language above

**Comment by bigtimebuddy** [comment]
Looks good! Thanks for the PR. I think a minor bump is appropriate for this.

**Comment by bigtimebuddy** [comment]
Good one, thanks!

**Pull request merged**